scripts/src/main/resources/scripts/command/java

```diff
@@ -53,7 +53,7 @@ function doSetup() {
     else
       os="linux"
     fi
-    local download_url="https://github.com/AdoptOpenJDK/openjdk${major_version}-binaries/releases/download/${jdk_folder}/OpenJDK${major_version}U-jdk_x64_${os}_hotspot_${software_version}${extension}"
+    local download_url="https://github.com/adoptium/temurin${major_version}-binaries/releases/download/${jdk_folder}/OpenJDK${major_version}U-jdk_x64_${os}_hotspot_${software_version}${extension}"
```

Comment:
Is this working for older versions or do we need a switch for that?

Reply:
Switch is required for older versions, working on it.

Reply:
done the changes